src/global/store/global.store.ts

```diff
@@ -18,13 +18,16 @@ export const useGlobalStore = defineStore('globalStore', {
       }
       this.messages.push(message);
       setTimeout(() => {
-        this.removeMessage(message.timestamp);
-      }, 5000);
+        this.removeMessage(message.id);
+      }, 25000);
     },
-    removeMessage(messageTimestamp: number): void {
-      this.messages = this.messages.filter(
-        (m) => m.timestamp !== messageTimestamp,
-      );
+    removeMessage(messageId: number): void {
+      this.messages = this.messages.filter((m) => m.id !== messageId);
```

Comment:
nie jestem fanem takich skrótów "m" niby fajnie to wygląda, skrótowo, ale jest zasadniczo zupełnie nieczytelne. lepiej po prostu message i tyle, czytelność, zrozumienie dla innych ponad zwięzłość moim zdaniem. 
Szczególnie w nazewnictwie.